### DISCUSSION TITLE
Css doesn't load

### DISCUSSION THREAD

**Discussion by JTzLinux** [community/community#23364] [Other Feature Feedback, Questions, & Ideas]
<p>Hello there everyone my CSS doesn’t load I saved it a path above under a folder called: “.css” in my code of the index.html I added: <code>&lt;link rel="stylesheet" href=".css/index.css"&gt;</code> I tried to force reload it with cleared cache without clearing cache I tried it on my local PC and it worked without any problems I tried to add a / before it and without it I added …/…/ without any results so I didn’t know anything else to do so might as well search an answer here</p>
<p>Thanks in advance</p>

**Comment by airtower-luna** [comment]
<p>In general a relative link will be better, because it should work both locally and in Pages. Exactly what is going on would require a look at your repository structure.</p>
<p>Things to look at:</p>
<ul>
<li>Was your attempt at relative paths correct? To refer to the parent directory use <code>../</code>, not <code>.../</code>.</li>
<li>Is capitalization correct? When people work on Windows the file system usually ignores it, but on the web it must be correct.</li>
</ul>

**Comment by JTzLinux** [comment]
<p>I tried …/…/ and …/…/ (2 dots) Yes capitalization is correct and I am an kali user not Windows</p>
<p>Edit: I just noticed github keeps on adding by the two dots another dot so I added: (2 dots)</p>

**Comment by airtower-luna** [comment]
<p><code>.../.../</code> will not work. “Parent directory” is <em>two</em> dots, not three. What else might be going wrong is impossible to say without a look at your repository, you’d need to share a link for that.</p>

**Comment by JTzLinux** [comment]
<p>Edit: Because I just noticed github made my first reply from two dots into three I will just say in the future: (2 Dots)</p>

**Comment by JTzLinux** [comment]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/JTzLinux/TestWiki">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/JTzLinux/TestWiki" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/60817679/181076548-40c398e3-1521-408d-9286-5a807d95788a.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/3/33c19f339241a4585133fdf5b93308b1df857adc_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/3/33c19f339241a4585133fdf5b93308b1df857adc_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/3/33c19f339241a4585133fdf5b93308b1df857adc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/3/33c19f339241a4585133fdf5b93308b1df857adc_2_10x10.png"></div>

<h3><a href="https://github.com/JTzLinux/TestWiki" target="_blank" rel="noopener nofollow ugc">GitHub - JTzLinux/TestWiki</a></h3>

  <p>Contribute to JTzLinux/TestWiki development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
I kept it simple for testing and already a simple line of code doesn’t work that great on github XD</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I see two issues there:</p>
<ol>
<li>
<p>The path to the CSS file is wrong. The <code>index.html</code> file currently includes <code>../../.css/index.css</code>, when from the directory layout it needs to be <code>.css/index.css</code> (because the <code>.css</code> folder is next to <code>index.html</code>).</p>
</li>
<li>
<p>The <code>.css</code> directory doesn’t make it to the deployed site. This is because of how Jekyll treats the <a href="https://jekyllrb.com/docs/structure/">Directory Structure</a>:</p>
</li>
</ol>
<blockquote>
<p>Every file or directory beginning with the following characters: <code>.</code> , <code>_ </code> , <code>#</code> or <code>~</code> in the <code>source</code> directory will not be included in the <code>destination</code> folder.</p>
</blockquote>
<p>Rename the <code>.css</code> directory to <code>css</code>, fix the relative path, and it should work. :slightly_smiling_face: Alternatively (for 2.) you could disable Jekyll for your site, but I think renaming is less confusing.</p>

**Comment by JTzLinux** [comment]
<p>Hello there airtower-luna,<br>
Thank you for your reply I will try it out as HTML dev it is just more easy for me to use .css because if I look via FTP on my server or Directly on my File explorer it will pop up as the first result I will try it out and remove characters</p>
<p>Edit: So I tried it and it seems to work</p>
